app/assets/stylesheets/static_pages.css.scss

```diff
@@ -2,6 +2,70 @@
 // // They will automatically be included in application.css.
 // // You can use Sass (SCSS) here: http://sass-lang.com/
 
+@import "./_variables";
+@import "./_mixins";
+
+$base-font-size: 10px;
+
+//convert px value to rem value
+@function px-to-rem($px, $base:$base-font-size) {
+  @return ($px / $base) * 1rem;
+}
+
+.site-section {
+  text-align: center;
+
+  h2 {
+    font-size: px-to-rem(36px);
+    margin-bottom: px-to-rem(30px);
+  }
+
+  p {
+    font-size: px-to-rem(18px);
+  }
+
+  a {
+    color: black;
+    @include arrow();
+  }
+
+  &.resources {
+    //resources-specific styles here
+    background-color: red; //temp color
+  }
+
+  &.community {
+    //community-specific styles here
+    background-color: green; // temp color
+  }
+
+  &.mentorship {
+    //mentorship-specific styles here
+    background-color: blue; // temp color
```

Comment:
These comments and temporary colors are fine for now. We'll finesse when everything is merged into the entry-page branch.